internal/pkg/agent/application/actions/handlers/handler_action_policy_change.go

```diff
@@ -189,8 +206,9 @@ func updateFleetConfig(log *logger.Logger, src remote.Config, dst *remote.Config
 
 	if src.Transport.Proxy.URL == nil ||
 		src.Transport.Proxy.URL.String() == "" {
-		log.Debug("proxy from fleet is empty or null, the proxy will not be changed")
+		log.Debugw("proxy from fleet is empty or null, the proxy will not be changed", "current proxy", dst.Transport.Proxy.URL)
```

Comment:
I believe `current proxy` becomes a key in the log message correct? That should not have a space in it.

```suggestion
		log.Debugw("proxy from fleet is empty or null, the proxy will not be changed", "current_proxy", dst.Transport.Proxy.URL)
```